content/rc/cloud-integrations/prometheus-integration.md

```diff
@@ -0,0 +1,169 @@
+
+{{< /note >}}
+
+## Quick start
+
+To get started with custom monitoring with Prometheus on Docker:
+
+1. To connect Prometheus and Grafana to your Redis Cloud databases, they must be running on an machine that:
+    - Is running on the same cloud provider as your Redis Cloud subscription
+    - Is located in the same region as your Redis Cloud subscription
+    - Is connected to the VPC subnet that is peered with your Redis Cloud subscription
+    - Allows outbound connections to port 8070, so that Prometheus can scrape the Redis Cloud server for data
+    - Allows inbound connections to port 9090 for Prometheus and 3000 for Grafana
+
+    Create a directory on this machine called `prometheus`.
+
+1. Within that directory, create a file called `prometheus.yml`.
+
+1. Add the following contents to `prometheus.yml`. Replace `<instance_ip>` with the IP address of the machine and `<cluster_name>` with the FQDN portion of your database endpoint.
+
+    ```yml 
+    global:
+        scrape_interval: 15s
+        evaluation_interval: 15s
+
+        # Attach these labels to any time series or alerts when communicating with
+        # external systems (federation, remote storage, Alertmanager).
+        external_labels:
+            monitor: "prometheus-stack-monitor"
+
+        # Load and evaluate rules in this file every 'evaluation_interval' seconds.
+        #rule_files:
+        # - "first.rules"
+        # - "second.rules"
+
+        scrape_configs:
+        # scrape Prometheus itself
+        - job_name: prometheus
+            scrape_interval: 10s
+            scrape_timeout: 5s
+            static_configs:
+                - targets: ["<instance_ip>:9090"]
+
+        # scrape Redis Cloud
+        - job_name: redis-cloud
+            scrape_interval: 30s
+            scrape_timeout: 30s
+            metrics_path: /
+            scheme: https
+            static_configs:
+                - targets: ["internal.<cluster_name>:8070"]
+    ```
+1. Set up your Prometheus and Grafana servers.
+
+    To set up Prometheus and Grafana on Docker containers:
+     1. Create a _docker-compose.yml_ file:
+
+        ```yml
+        version: '3'
+        services:
+            prometheus-server:
+                image: prom/prometheus
+                ports:
+                    - 9090:9090
+                volumes:
+                    - ./prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
+
+            grafana-ui:
+                image: grafana/grafana
+                ports:
+                    - 3000:3000
+                environment:
+                    - GF_SECURITY_ADMIN_PASSWORD=secret
+                links:
+                    - prometheus-server:prometheus
+        ```
+
+    1. To start the containers, run:
+
+        ```sh
+        $ docker compose up -d
+        ```
+
+    1. To check that all the containers are up, run: `docker ps`
```

Comment:
1. Run `docker ps` to check if all the containers are up and running. 

Whenever possible, try to start action steps with a verb. 
Are they looking for a certain status in the command output such as "running" or "started." If so we should mention that here.